nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/controller/service/ServiceStateTransition.java

```diff
@@ -58,6 +65,14 @@ public boolean enable() {
             state = ControllerServiceState.ENABLED;
 
             enabledFutures.forEach(future -> future.complete(null));
+
```

Comment:
Maybe some debug logging here?

Reply:
What are you thinking makes sense to log at a debug level?

Reply:
i take it back.